src/controller.coffee

```diff
@@ -827,8 +827,11 @@ Editor::redrawPalette = ->
 
     paletteBlockView.group.element.setAttribute 'data-id', entry.id
 
-    # enable tooltips
-    paletteBlockView.group.element.setAttribute('class', 'droplet-hover-div')
+    # enable tooltips, making sure not to overwrite any additional classes
+    # that may have been added (e.g. by tooltipster)
+    paletteBlockClass = paletteBlockView.group.element.getAttribute('class')
+    unless paletteBlockClass && paletteBlockClass.includes('droplet-hover-div')
+      paletteBlockView.group.element.setAttribute('class', 'droplet-hover-div')
```

Comment:
If the element has just the class name from tooltipster, and does not have 'droplet-hover-div', won't we still blow away the other class?

Should the last line instead be ` paletteBlockView.group.element.setAttribute('class', paletteBlockClass + ' droplet-hover-div')`